src/applications/personalization/dashboard/components/ManageYourVAHealthCare.jsx

```diff
@@ -106,14 +145,18 @@ const ManageYourVAHealthCare = ({
     >
       <PrescriptionsWidget />
     </DowntimeNotification>
-    {isEnrolledInHealthCare && <ScheduleAnAppointmentWidget />}
+    {isEnrolledInHealthCare &&
+      !isCernerOnlyPatient && <ScheduleAnAppointmentWidget />}
+    {isEnrolledInHealthCare &&
+      isCernerOnlyPatient && <ScheduleAnAppointmentCernerWidget />}
   </>
 );
 
 const mapStateToProps = state => {
   const isEnrolledInHealthCare = isEnrolledInVAHealthCare(state);
   const hcaEnrollmentStatus = selectEnrollmentStatus(state);
   const showServerError = hasESRServerError(state);
+  const isCernerOnlyPatient = selectIsCernerOnlyPatient(state);
```

Comment:
We need to confirm that this is only being shown to Vets who are _only_ patients in Cerner facilities vs. showing it to Vets as long as they are a patient in at least one Cerner facility.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every

every makes sure that all of the facilities must meet the `isCerner` condition

Reply:
I know but what hasn't been made clear is if we only want to show it to Vets who meet that condition or if it's okay to show it if at least one of a Vet's facilities is a Cerner facility.

Reply:
@erikphansen added new selector